config/enderio/enderiomachines.cfg

```diff
@@ -149,20 +149,20 @@ generator {
         B:helmetChargeOthers=true
 
         # Energy generated per TICK by Simple Photovoltaic Panels. (synced from server) [range: 1 ~ 2147483647, default: 10]
-        I:solarPanel0Gen=10
+        I:solarPanel0Gen=200
```

Comment:
The Solar panels have been buffed too much imo, I would prefer the values to be:
* 50
* 200
* 800
* 3200

I know they're expensive, but they are fully passive and very easily stackable.